docs/guides/migrating.md

```diff
@@ -121,7 +121,7 @@ While the `style` utility from v4 should continue to work as expected, you can t
 
 ```js
 // v4
-const transition = style({
+const transition = styled({
```

Comment:
This is the v4 `style` API, so I think what's currently shown is correct

Reply:
You’re right, my bad